data-prepper-plugins/aws-plugin-api/src/main/java/org/opensearch/dataprepper/aws/api/AwsCredentialsOptions.java

```diff
@@ -49,6 +54,10 @@ public static AwsCredentialsOptions defaultOptions() {
         return DEFAULT_OPTIONS;
     }
 
+    public static AwsCredentialsOptions defaultOptionsWithDefaultCreds() {
```

Comment:
```suggestion
    public static AwsCredentialsOptions defaultOptionsWithDefaultCredentialsProvider() {
```

Reply:
Thanks @dlvenable. I have made the name change.